index.html

```diff
@@ -1,55 +1,243 @@
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h2 class="title">Multi-Post Stories Gain+Glory</h2>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="third-section">
+      <div class="container">
+        <img class="frameleft" src="./Images/frameleft.svg" alt="">
+        <img class="frameright" src="./Images/frameright.svg" alt="">
+        <h2>About Me</h2>
+        <p class="about-desc">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="box-button about-btn">Get My Resume</button>
+      </div>
+      <ul class="boxes">
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon1.svg" alt="" /></div>
+          <div class="about-menu">
+            <h2 class="about-title">Languages</h2>
+            <ul class="about-list">
+              <li class="about-list-item">Javascript</li>
+              <li class="about-list-item">Ruby</li>
+              <li class="about-list-item">Html</li>
+              <li class="about-list-item">CSS</li>
+            </ul>
+          </div>
+        </li>
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon2.svg" alt="" /></div>
+          <div class="about-menu">
+            <h2 class="about-title">Frameworks</h2>
+            <ul class="about-list">
+              <li class="about-list-item">Bootstrap</li>
+              <li class="about-list-item">Ruby on rails</li>
+              <li class="about-list-item">RSpec</li>
+              <li class="about-list-item">Capybara</li>
+              <li class="about-list-item">Selenium</li>
+            </ul>
+          </div>
+        </li>
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon3.svg" alt="" /></div>
+          <div class="about-menu">
+            <h2 class="about-title">Skills</h2>
+            <ul class="about-list">
+              <li class="about-list-item">Codekit</li>
+              <li class="about-list-item">Github</li>
+              <li class="about-list-item">Codepen</li>
+              <li class="about-list-item">Gitlab</li>
+              <li class="about-list-item">Terminal</li>
+            </ul>
+          </div>
+        </li>
```

Comment:
- [ ] As previous reviewers indicated, please try to keep your design as close as possible the figma's version. Kindly consider these looks are inspect to find variances in the area of `box-shadow` , `font-size` , `font-family`, `font-weight`, `padding` and `margin`:
**Figma**
![image](https://user-images.githubusercontent.com/99715640/212430450-c9da512d-793d-4a45-a7da-1bb389541d9f.png)

**Your design**
![image](https://user-images.githubusercontent.com/99715640/212430928-01cbe6b7-6254-48ce-b477-5311d9eba843.png)

_**Note**_: I have tried to show you the appearance of your page on a device of `width:  375px` which is what this project requires.